figma-screenshots/AdminFeedbackViews/README.md

```diff
@@ -0,0 +1,19 @@
+# Admin View of Aggregated Mentor Feedback
+
+To allow administrator users to see what feedback mentors are receiving from mentees, we created two Figma mocks.
+
+The initial wireframe included stars and showed both views.
+
+<img title="Initial-Feedback-Wireframe" alt="A wireframe with stars and two tables with three rows and columns each" src="./Initial-Feedback-Wireframe.png">
+
+Here's the first Figma mock showing the Mentor Feedback view where mentors are listed one per row with a summary of the feedback they have received from mentees.
+
+<img title="Mentor-Feedback" alt="A Figma mock with a table that has four rows and three columns" src="./Mentor-Feedback.png">
+
+Here's the second Figma mock showing the Mentor Modal view where a modal pops up when the administrator clicks on one mentor row in the Mentor Feedback view. The modal displays a table with three columns and as many rows as needed (one per mentee review) with each row representing one mentee review of that mentor.
+
+<img title="Mentor-Modal" alt="A Figma mock with a table that has four rows and three columns" src="./Mentor-Modal.png">
+
+On the initial wireframe and Figma mocks, stars were included in the form of a three-star system with orange stars to approximately match the color of the UD logo. The plan was to use the filling of those stars to represent the overall sentiment analysis returned from DS as one of three words. "Negative" would be represented as one filled-in orange star out of three, "neutral" as two, and "positive" as all three filled. In response to negative feedback from both stakeholder Jason and DS manager Robert at the stakeholder meeting on 05 OCT 2022, the middle column of stars was removed and replaced with simple text. Therefore, stars are no longer present in the final Figma mocks.
+
+In addition, stakeholder Jason's feedback suggesting alternating background colors on the rows was implemented to allow the user to clearly delineate where one row (and especially block of text) ends and another begins.
```

Comment:
I like the design and adding different colors for each row was definitely a good idea.

Reply:
I also like the different colors. Well done!